pages.ru/common/hunspell.md

```diff
@@ -0,0 +1,16 @@
+# hunspell
+
+> Проверка орфографии.
+> Больше информации: <https://hunspell.github.io/>.
+
+- Проверить орфографию в указанном файле:
+
+`hunspell {{путь/до/файла}}`
+
+- Проверить орфографию в указанном файле, используя американский словарь (en_US):
+
+`hunspell -d {{en_US}} {{путь/до/файла}}`
```

Comment:
Just note that we prefer long options. Maybe the original page should be updated if the corresponding long option is available. :) The same applies to almost all other code samples.

Reply:
I remember using this command before. Long arguments aren't present in it. I will check it once again. Yep, it only has short arguments. [Ref](https://github.com/hunspell/hunspell)